### PR TITLE
GitHub Actions workflows for main branch.

### PR DESCRIPTION
**Pull request by isarkis**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@
 /venv/*
 _certs/
 .coverage
+mobile_harness_gateway_pb2.py
```
Comment (kaidokert) [inline .gitignore:12]: Should these be only in tools/ directory ?

File: docker/windows/base/build/Dockerfile [hunk 2]
```diff
@@ -32,7 +32,9 @@ COPY ./list_python_processes.py /list_python_processes.py
 # Install deps via chocolatey.
 RUN iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'));`
     mkdir C:\choco-cache;`
+    # Install Python 3 before Python 2, so the default on the path is python2.
     choco install -y -c C:\choco-cache python3 -ia '/quiet InstallAllUsers=1 PrependPath=1 TargetDir="C:\Python3"';`
+    choco install -y -c C:\choco-cache python2 --params '/InstallDir:C:\python2';`
```
Comment (kaidokert) [inline docker/windows/base/build/Dockerfile:37]: Is this still needed ?

Comment (isarkis) [reply docker/windows/base/build/Dockerfile]: I don't python2 is needed, let's rebuild the runner image and deploy to GKE...

File: .gitignore (hunk 1 above)
Comment (isarkis) [reply .gitignore:12]: @iamjazzar , can you take a look?

Comment (isarkis) [reply .gitignore]: I've temporarily removed all mobile harness references from this PR.

**Pull request merged**